packages/terra-profile-image/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Changelog
 Unreleased
 ----------
 
+### Changed
+* Changed Src to be an optional prop.
```

Comment:
Let's maybe do something like - 

1. Changed `src` to be an optional prop.
2. Use the avatar image as the default fallback image when `src` is not provided.